### PR TITLE
Remove legacy automation

### PR DESCRIPTION
**Pull request by aesteve-rh**
Rename automation folder to ci and update all
references to automation scripts.

Remove unused ci (old automation) scripts.

Fixes: https://github.com/oVirt/vdsm/issues/43
Signed-off-by: Albert Esteve <aesteve@redhat.com>

### CODE REVIEW

**Review by nirs — CHANGES_REQUESTED** [review]
Why rename tons of files in automation/ and delete them in the next commit? It would be better to delete the unused file first, then rename the few used files.

File: doc/development.md [hunk 1]
```diff
@@ -26,7 +26,7 @@ Clone your fork:
 
 Install additional packages for Fedora, CentOS, and RHEL:
 
-    sudo dnf install -y `cat ci/check-patch.packages`
+    ci/install-pkg.sh
```
Comment (nirs) [inline doc/development.md:29]: This is not used in the ci - it is used only for preparing development environment manually, so why does it belong in the ci directory?

Comment (aesteve-rh) [reply doc/development.md]: right, moved to `contrib`, it should pertain there better.

**Review by mz-pdm — APPROVED** [review]

File: doc/development.md [hunk 2]
```diff
@@ -26,7 +26,7 @@ Clone your fork:
 
 Install additional packages for Fedora, CentOS, and RHEL:
 
-    sudo dnf install -y `cat ci/check-patch.packages`
+    contrib/install-pkg.sh
```
Comment (nirs) [inline doc/development.md:29]: contrib/install-pkg.sh is not the best name, and the script does have any documentation so someone looking in this script will have no idea what is the purpose of it.

**Review by nirs — APPROVED** [review]

File: doc/development.md (hunk 2 above)
Comment (aesteve-rh) [reply doc/development.md:29]: Good idea: https://github.com/oVirt/vdsm/issues/309

I can tackle this in a followup PR so that we unblock the SPDX branch.

**Pull request merged**